opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -446,6 +446,29 @@ message HistogramDataPoint {
   // Flags that apply to this specific data point.  See DataPointFlags
   // for the available flags and their meaning.
   uint32 flags = 10;
+
+  // Note: the definition used to for min and max fields below is
+  // based on temporality in a way meant to match the Prometheus
+  // Summary data type used with quantiles 0 and 1, respectively, when
+  // paired with cumulative temporality.  This uses a fixed width that
+  // SHOULD be up to 10 minutes or for as long as data is available.
+  // https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#summary
+  
+  // min is the minimum value over a recent window of time.
+  //
+  // The interpretation is based on Metric's temporality.
+  // - delta temporality: the min value MUST be computed over [start_time, end_time]
+  // - cumulative temporality: the min value SHOULD be computed over [now - WIDTH, end_time]
+  //   for WIDTH up to 10 minutes.
+  optional double min = 11;
```

Comment:
FYI the use of "optional" will make the collector impossible to upgrade to this version right away (will take couple of months probably). Not suggesting to not do the correct thing, but setting the right expectations.

Reply:
This was discussed and accepted. We think it is worth handling "optional" fields.